Updates route-level pass id headers setting (#1226)
pomerium/documentation: content/docs/reference/routes/pass-identity-headers.mdx

## Summary

When set to true, the **Pass Identity Headers (per route)** setting will pass identity headers to the upstream application.

> If you enable the [global-level Pass Identity Headers](/docs/reference/pass-identity-headers) setting, the route-level setting will still take precedence.

Reviewer: What do you think about something like this?

```suggestion
The default value for this setting is controlled by the [global-level Pass Identity Headers](/docs/reference/pass-identity-headers) setting.
```
